tests/jsonrpc_calls.rs

```diff
@@ -0,0 +1,37 @@
+extern crate env_logger;
+extern crate jsonrpc_client;
+extern crate jsonrpc_core;
+extern crate jsonrpc_http_server;
+extern crate spectral;
+
+use jsonrpc_client::*;
+use jsonrpc_core::*;
+use jsonrpc_http_server::*;
+use spectral::prelude::*;
+
+fn main() {}
+
+#[test]
+
+fn get_server_response() {
+    let _ = env_logger::try_init();
+
+    let mut io = IoHandler::new();
+    io.add_method("helloworld", |_: Params| {
+        Ok(Value::String("hello".to_string()))
+    });
+    let server = ServerBuilder::new(io)
+        .start_http(&"127.0.0.1:3030".parse().unwrap())
+        .unwrap();
+    let client = RpcClient::new(HTTPClient::new(), "http://127.0.0.1:3030");
+
+    let res =
+        client.send::<String, ()>(&RpcRequest::new0(JsonRpcVersion::V2, "test", "helloworld"));
+
+    assert_that(&res)
```

Comment:
I don't think you are testing the log feature.

Reply:
Yep, there is no test for the log feature: should I add a test for it? This is just a basic test that checks if the request and response match.

Reply:
How would you test logging? I will assume you are serious about your question so I'll give a serious answers:

I think testing logs is pretty pointless because they are so simple. You can verify that they work by seeing that they are there. Formatting/serializing is a different story but that is also not necessary here.

I don't see a point in adding a test for a log (how would you even do that?)

Reply:
Yeah, that's what I thought. Thanks for the explanation!